test/form-data.js

```diff
@@ -1,8 +1,6 @@
 import {FormData as FormDataNode} from 'formdata-node';
-import {FormData} from 'formdata-polyfill/esm.min.js';
-import {Blob} from 'fetch-blob/from.js';
 import chai from 'chai';
-import {Request, Response} from '../src/index.js';
+import {Request, Response, FormData, Blob} from '../src/index.js';
```

Comment:
Not really related, but I think that this can import from `node-fetch` with ESM, which will import the local package:

```suggestion
import {Request, Response, FormData, Blob} from 'node-fetch';
```

Reply:
umm, i think i rather prefer relative import
Thinking that it's easier for the path resolver to quicker lookup the file/module rather than going via some package.json lookup

Reply:
I can't imagine that would even make a milliseconds difference? The nice thing about using `from 'node-fetch'` is that you also test that the public importable API is how you expect it. E.g. it tests that your `exports` is setup correctly ☺️

Reply:
"många bäckar små" as we say in Sweden... for one file it might not make a differences. but when when you have many dependencies like [this](https://npmgraph.js.org/?q=xo) then it adds up. I bet if every file actually used the extension in each and every file then it would not have to 2nd guess what you meant to import.

relative import is also more remote friendlier as it would not have access to the filesystem to scan for things. then you would need import-maps

https://nodejs.org/dist/latest-v17.x/docs/api/modules.html#all-together
https://github.com/nodejs/node/blob/34e3dd50348dd824c881c183824237532c7721c5/lib/internal/modules/cjs/loader.js#L346

I know that unix is quite good at handling many small files and stat-ing each and every thing relative fast but windows might struggle with [the cost of many small modules](https://nolanlawson.com/2016/08/15/the-cost-of-small-modules/)